adam-apis/src/test/scala/org/bdgenomics/adam/apis/java/JavaADAMContextSuite.scala

```diff
@@ -20,12 +20,17 @@ package org.bdgenomics.adam.apis.java
 import org.apache.spark.rdd.RDD
 import org.apache.spark.api.java.JavaRDD._
 import org.apache.spark.api.java.JavaRDD
+import org.bdgenomics.adam.rdd.ADAMContext
 import org.bdgenomics.adam.rdd.ADAMContext._
 import org.bdgenomics.adam.util.ADAMFunSuite
 import org.bdgenomics.formats.avro.AlignmentRecord
 
 class JavaADAMContextSuite extends ADAMFunSuite {
 
+  sparkTest("ctr is accessible") {
```

Comment:
I agree, but if we change visibility again, at least something will fail here instead of in a downstream project.